README.md

```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
 const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
+
+
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
+
+-  Are there character limits for error messages?
+-  Addressing messaging around fields set via data hook (ex: we have a data hook that populates an empty field. Upon landing in review, that field is populated with a value and a message.  If another field on that record is populated, when data hooks are run again the field remains populated but without an error message letting the user know the field had been initially transformed.
```

Comment:
Add this to the FAQ for now.

This hints at the DataHook lifecycle, which is implied right now, but should be made more clear.